README.markdown

```diff
@@ -51,23 +50,30 @@ registry_value { 'HKLM\System\CurrentControlSet\Services\Puppet\Description':
 }
 ```
 
+### Manage a single Registry value with a complex name
+
+``` puppet
+registry_value { 'ComplexValue':
+  path       => 'HKLM\System\CurrentControlSet\Services\Puppet\\\ValueWithA\Backslash',
+  ensure     => present,
+  type       => string,
+  data       => "The Puppet Agent service periodically manages your configuration",
+}
+```
+
 ### Manage a Registry value and its parent key in one declaration
 
 ``` puppet
 class myapp {
   registry::value { 'puppetmaster':
-  key  => 'HKLM\Software\Vendor\PuppetLabs',
-  data => 'puppet.puppet.com',
+    key  => 'HKLM\Software\Vendor\PuppetLabs',
+    data => 'puppet.puppet.com',
   }
 }
 ```
 
 Puppet looks up the key 'HKLM\Software\Vendor\PuppetLabs' and makes sure it contains a value named 'puppetmaster' containing the string 'puppet.puppet.com'.
```

Comment:
Should we backtick these (like the rest of the file / examples)?

Reply:
single quotes for strings.  Backticks for lables/symbols